src/UniversalSentenceEncoder/index.js

```diff
@@ -29,45 +44,66 @@ class UniversalSentenceEncoder {
   }
 
   /**
-   * Encodes a string or array based on the USE
-   * @param {*} textString 
-   * @param {*} callback 
+   * @public
+   * @param {(string | string[])} text
+   * @param {function} callback
+   * @return {Promise}
    */
-  predict(textArray, callback){
-    return callCallback(this.predictInternal(textArray), callback);
+  predict(text, callback){
+    return callCallback(this.predictInternal(text), callback);
   }
 
-  async predictInternal(textArray){
-    try{
-      const embeddings = await this.model.embed(textArray);
+  /**
+   * @private
+   * @param {(string | string[])} text
+   * @return {Promise<number[][]>}
+   */
+  async predictInternal(text){
+    try {
+      const embeddings = await this.model.embed(text);
       const results = await embeddings.array();
       embeddings.dispose();
       return results;
-    } catch(err){
+    }
+    // TODO: it would be better not to catch the error
```

Comment:
nonblocking comment/ idea:
* This is an interesting note about how errors should be handled and by whom 🤔 Definitely something to chew on cc'ing @bomanimc @tlsaeger @shiffman 
* Thanks for pointing this out!